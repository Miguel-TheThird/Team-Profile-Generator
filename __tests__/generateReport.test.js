const generateReport = require(`../lib/generateReport`)

describe("HTML report", () => {
    it("Check if any change was made in HTML file", () => {
        expect(generateReport()).toMatchSnapshot();
    }

)})