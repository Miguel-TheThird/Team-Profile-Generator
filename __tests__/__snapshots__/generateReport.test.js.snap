// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HTML report Check if any change was made in HTML file 1`] = `
"<!DOCTYPE html>
<html lang=\\"en\\">

<head>
  <title>Flexbox</title>
  <meta charset=\\"UTF-8\\" />
  <meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1\\" />
  <link rel=\\"stylesheet\\" type=\\"text/css\\" href=\\"./assets/style.css\\" />
</head>

<body>
  <header class=\\"main-header\\">
    <h1>My team</h1>
  </header>

  <main>
    <div class = \\"parentDiv\\" >
      <div class=\\"parentCard\\">
        <!--Manager cards -->
      </div>
      <br>
      <div class=\\"card\\">
        <!--Engineer cards -->
      </div>
      <br>
      <div class=\\"card\\">
        <!--Intern cards -->
      </div>
    </div>
  </main>
</body>

</html>"
`;
